spec/react_on_rails/generators/install_generator_spec.rb

```diff
@@ -60,11 +60,11 @@
       expect(GeneratorMessages.output).to include(expected)
     end
 
-    specify "react with redux generator contains a helpful message" do
-      run_generator_test_with_args(%w[--redux], package_json: true)
-      # GeneratorMessages.output is an array with the git error being the first one
-      expect(GeneratorMessages.output).to include(expected)
-    end
+    # specify "react with redux generator contains a helpful message" do
+    #   run_generator_test_with_args(%w[--redux], package_json: true)
+    #   # GeneratorMessages.output is an array with the git error being the first one
+    #   expect(GeneratorMessages.output).to include(expected)
+    # end
```

Comment:
why leave a commented out test? why comment out?

Reply:
Oh! This is is mistake. I did this to get the local test run faster.